nservicebus/ravendb/index.md

```diff
@@ -11,12 +11,7 @@ redirects:
  - nservicebus/ravendb/connecting
 ---
 
-Uses the [RavenDB document database](http://ravendb.net/) for storage.
-
-
-## NServiceBus 5 and above
-
-RavenDB is no longer merged into the core. The RavenDB-backed persistence is available as a separate [NuGet package](https://www.nuget.org/packages/NServiceBus.RavenDB). This allows NServiceBus and RavenDB to be upgraded independently.
+Uses the [RavenDB document database](http://ravendb.net/) for storage. Starting with NServiceBus 5, the RavenDB-backed persistence is no longer ILMerged into the Core and is available as a separate [NuGet package](https://www.nuget.org/packages/NServiceBus.RavenDB), which allows NServiceBus and RavenDB to be upgraded independently. See below for information specifically related to NServiceBus 3 and NServiceBus 4. 
```

Comment:
> Starting with NServiceBus 5, 

See the version specific language here:
https://github.com/particular/docs.particular.net#version-language

So, maybe `When using NServiceBus Versions 5 and above`?